twilight-model/src/guild/onboarding/mode.rs

```diff
@@ -0,0 +1,95 @@
+use std::fmt::{Display, Formatter, Result as FmtResult};
```

Comment:
if the other models arent moved into this crate, this should be moved to the http crate

Reply:
This enum is still used in the model crate.